### DISCUSSION TITLE
migrate to latest .Net

### DISCUSSION THREAD

**Discussion by draylegend** [PenguLoader/PenguLoader#102] [Ideas]
I get the following warning by opening `PenguLoader`

![image](https://github.com/PenguLoader/PenguLoader/assets/16114644/666d3cec-7ad7-416d-bf62-7f1b6efdac51)

@nomi-san do you plan to migrate to the latest .Net version?

**Comment by BakaFT** [comment]
We stay dotnet framework 4.5 for Windows 7 compatibility.

🤔I know there is Native AOT in dotnet 8. But if it takes lots of code changes, that would be hard to. You know, this is open-source.

**Comment by nomi-san** [comment]
Framework version does not matter, but we should keep the app size is smallest possible.
.NET Framework 4.5 is built-in since Windows 7, while .NET 8 Native AOT does not support completely trimming Winforms and WPF.
